.config/cargo_spellcheck.dic

```diff
@@ -1,4 +1,5 @@
 80
+0x00
```

Comment:
I am very very unsure about this ... better wrap it in `0x00` where you need it.

Reply:
I wasn't sure how else to satisfy the spellchecker 🤷 

I'm open to other suggestions though